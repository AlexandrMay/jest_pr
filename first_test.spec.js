const axios = require('axios');
axios.defaults.baseURL = 'http://5.189.186.217/';

let token = null;

describe("My first suite", () => {
  it("Authorization", async () => {
    const response = await axios.post("api/auth/login", {
      headers: {
        "Content-Type": "application/json"
      },
      email: "maysalexandr@gmail.com",
      password: "123456"
    });
    token = response.data.token;
    expect(response.data).toMatchSnapshot({
      token: expect.any(String)
    });
  });
});
