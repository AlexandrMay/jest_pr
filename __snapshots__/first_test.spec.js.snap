// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`My first suite Authorization 1`] = `
{
  "token": Any<String>,
}
`;
